homework3/src/main/java/first/FirstTask.java

```diff
@@ -0,0 +1,26 @@
+package first;
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
+
+public class FirstTask {
+
+    public static void main(String[] args) {
+
+        Random random = new Random();
+        Set<String> stringSet = new HashSet<>();
+        for (int i = 0; i < 5; i++) {
+            StringBuilder sb = new StringBuilder();
+            int len = random.nextInt(10) + 6;
+            while (len-- > 0) {
+                char c = (char) (random.nextInt(26) + 'a');
+                sb.append(c);
+            }
+            stringSet.add(sb.toString());
+
+        }
```

Comment:
that should be in separate method
so in main we should do just 2 things:
1. List<String> randomStringList = randomStringsGeneration() //<--plus don't forget to add the parameter for the number of strings that we want to create
2. sout